src/OrchardCore.Modules/OrchardCore.ContentFields/Assets/trumbowyg.media.tag.js

```diff
@@ -20,7 +20,7 @@
                             $('#mediaHtmlFieldSelectButton').on('click', function (v) {
                                 var mediaBodyContent = "";
                                 for (i = 0; i < mediaApp.selectedMedias.length; i++) {
-                                    mediaBodyContent += ' {{ "' + mediaApp.selectedMedias[i].mediaPath + '" | asset_url | img_tag }}';
+                                    mediaBodyContent += ' [media]' + mediaApp.selectedMedias[i].mediaPath + '[/media]';
```

Comment:
The media app now only returns safe code tags, not liquid tags.

Did consider this could be an settings option, but felt it was better to just migrate away.